flybywiresim/aircraft
# fix: apu fuel consumption without left inner pumps

Fixes #8146

## Summary of Changes

APU pump is of type "APUDriven" which only works once the APU is actually running (gravity feeding to APU and engines does not work in MSFS yet, only from tanks to tanks). Changed it to type electric, so it works without it running and without the need to enable the left inner tank pumps as a workaround.

Also adapted the logic to start/stop the MSFS apu a bit, as `APU SWITCH` seemed a bit flaky to me, causing the APU to start and stop all the time.


## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Turn on the APU without turning on any of the fuel pumps.
2. Open SD Fuel page and observe left inner tank level going down (it takes a while as the APU does not consume that much fuel)
3. (Optional) Use Simvarwatcher to verify fuel flow in FUEL LINE 18 is present.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
